src/hooks/useLogin.ts

```diff
@@ -71,10 +71,18 @@ const useLogin = () => {
 			data: request,
 		});
 
-		// API 수정 필요
-		toast(`${response.result}`);
+		console.log(response);
+
+		if (response.isSuccess) {
+			if (response.result.isExisting) {
+				toast(response.result.message);
+				return true;
+			}
```

Comment:
```suggestion
			toast(response.result.message);
			return response.result.isExisting;
```

현재는 이메일이 중복되지 않는 경우만 메세지를 띄우고 있는데, 이메일이 중복되는 경우에도 사용자가 이미 사용 중인 메시지라 사용할 수 없다는 문구를 확인할 수 있도록 하는 건 어떨까요?